portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,164 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+*{
+    box-sizing: border-box;
 }
 
-#greeting-container {
-  margin-top: 20px;
+nav {
+    background-color: black;
+    color: white;
+    text-align: center;
+    font-size: 24px;
+    width: 100%;
+    overflow: hidden;
+    position: sticky;
 }
+
+nav span {
+    float: left;
+    padding: 14px 16px;
+    color: white;
+}
+
+nav a {
+    color: white;
+    text-decoration: none;
+    padding: 14px 16px;
+    float: right;
+
+}
+
+nav a:hover {
+    /* for underline under anchor text */
+    /* text-decoration: underline;
```

Comment:
Remove commented out rules.

Reply:
Sorry i first used an underline with the hover effect for the navbar but then i decided to do something else.